### PR TITLE
Extend ptp4l template

### PR DESCRIPTION
**Pull request by gerhard-tinned**
Extended the template for ptp4l with the following variables:

* delay_mechanism
* time_stamping

### CODE REVIEW

File: manifests/ptp4l.pp [hunk 1]
```diff
@@ -21,6 +23,8 @@
   validate_numeric($hybrid_e2e, 1, 0)
   validate_re($network_transport, ['UDPv4', 'UDPv6', 'L2'], "Parameter 'network_transport' must be one of 'UDPv4', 'UDPv6' or 'L2'")
   validate_re($clock_servo, ['pi', 'linreg', 'ntpshm', 'nullf'], "Parameter 'clock_servo' must be one of 'pi', 'linreg', 'ntpshm' or 'nullf'")
+  validate_re($delay_mechanism, ['E2E', 'P2P'], "Parameter 'delay_mechanism' must be one of 'E2E' or 'P2P'")
+  validate_re($time_stamping, ['hardware', 'software'], "Parameter 'delay_mechanism' must be one of 'hardware' or 'software'")
```
Comment (lukebigum) [inline manifests/ptp4l.pp:27]: The validate_re error message here still says "delay_mechanism" instead of "time_stamping".

**Review by lukebigum — CHANGES_REQUESTED** [review]
The validate_re error message here still says "delay_mechanism" instead of "time_stamping".

**Comment by lukebigum** [comment]
Other than the error message, it looks fine. I'll spend a bit of time getting the actual tests to pass in Travis tomorrow.

**Comment by gerhard-tinned** [comment]
Thanks for fixing the mistake in the error message.

**Pull request merged**